sno/structure.py

```diff
@@ -76,7 +84,7 @@ def fast_import_tables(
             )
             p.stdin.write(header.encode("utf8"))
 
-            if not repo.is_empty:
+            if head_tree is not None:
                 # start with the existing tree/contents
                 p.stdin.write(b"from refs/heads/master^0\n")
```

Comment:
Maybe put a TODO next to this refs/heads/master thing because, we probably shouldn't always be importing to master. Don't need to fix it in this PR but worth flagging.